.ci/oci-stable-all-namespaces.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+export OPERATOR_REPO=$(dirname $(dirname $(readlink -f "$0")));
+source "${OPERATOR_REPO}"/.github/bin/common.sh
+source "${OPERATOR_REPO}"/.github/bin/oauth-provision.sh
+
+#Stop execution on any error
+trap "catchFinish" EXIT SIGINT
+
+overrideDefaults() {
+  export DEV_WORKSPACE_ENABLE="true"
+}
+
+runTests() {
```

Comment:
There is no hints (except file name) that it is about deploying Eclipse Che from `stable-all-namespaces` channel.
How does it work?

Reply:
Right, my mistake. I will add comments that Openshift CI scripts will create the catalogSource for the new channel and subscriptions using Openshift CI workflow: https://steps.ci.openshift.org/reference/optional-operators-subscribe